scripts/runtestsuite/runtestsuite.py

```diff
@@ -237,16 +239,28 @@ def report_detailed_failure(self, scene, reference_filepath, output_filepath, lo
 
     def __write_header(self, args):
         script_path = os.path.realpath(__file__)
+        git_hash =  subprocess.check_output(['git', 'rev-parse', 'HEAD'])
+        git_title = subprocess.check_output(['git', 'rev-parse', '--abbrev-ref', 'HEAD'])
 
         self.file.write(self.__render(self.header_template,
                                       {'test-date': CURRENT_TIME,
                                        'python-version': utils.get_python_version(),
                                        'script-path': script_path,
-                                       'script-version': VERSION,
+                                       'script-version': "HELLO WORLD", #VERSION,
                                        'appleseed-binary-path': args.tool_path,
                                        'max-abs-diff-allowed': VALUE_THRESHOLD,
-                                       'max-diff-comps-count-allowed': MAX_DIFFERING_COMPONENTS}))
+                                       'max-diff-comps-count-allowed': MAX_DIFFERING_COMPONENTS,
+                                       'git-hash': git_hash,
+                                       'git-title': git_title}))
         self.file.flush()
+    
+    def __write_stats(self, total_time, success, failures):
+        success_rate = "{0}%".format(success)
+        failures_str = "{0} out of {1} test scene(s)".format(failures[0], failures[1])
```

Comment:
Minor styling issue : here you use the prefix `_str`, but not everywhere else. You can remove that prefix to have a more consistent code base :)